cfg/1.1.0/definitions.yaml

```diff
@@ -10,252 +10,251 @@ groups:
+          set: true
+      remediation: |
+        Edit or create the file `/etc/modprobe.d/CIS.conf` and add the following line:
+        install jffs2 /bin/true
+        Run the following command to unload the `jffs2` module:
+        # rmmod jffs2
+      scored: true
+    - id: 1.1.1.3.b
+      description: "Ensure mounting of jffs2 filesystems is disabled"
+      audit: "lsmod | grep jffs2"
+      tests:
+        test_items:
+        - flag: ""
+          compare:
+            op: eq
+            value: ""
+          set: true
+      remediation: |
+        Edit or create the file `/etc/modprobe.d/CIS.conf` and add the following line:
+        install jffs2 /bin/true
+        Run the following command to unload the `jffs2` module:
+        # rmmod jffs2
+      scored: true
+    - id: 1.1.1.4.a
+      description: "Ensure mounting of hfs filesystems is disabled"
+      audit: "modprobe -n -v hfs"
+      tests:
+        test_items:
+        - flag: "install /bin/true"
+          set: true
+      remediation: |
+        Edit or create the file `/etc/modprobe.d/CIS.conf` and add the following line:
+        install hfs /bin/true
+        Run the following command to unload the `hfs` module:
+        # rmmod hfs
+      scored: true
+    - id: 1.1.1.4.b
+      description: "Ensure mounting of hfs filesystems is disabled"
+      audit: "lsmod | grep hfs"
+      tests:
+        test_items:
+        - flag: ""
+          compare:
+            op: eq
+            value: ""
+          set: true
+      remediation: |
+        Edit or create the file `/etc/modprobe.d/CIS.conf` and add the following line:
+        install hfs /bin/true
+        Run the following command to unload the `hfs` module:
+        # rmmod hfs
+      scored: true
+    - id: 1.1.1.5.a
+      description: "Ensure mounting of hfsplus filesystems is disabled"
+      audit: "modprobe -n -v hfsplus"
+      tests:
+        test_items:
+        - flag: "install /bin/true"
+          set: true
+      remediation: |
+        Edit or create the file `/etc/modprobe.d/CIS.conf` and add the following line:
+        install hfsplus /bin/true
+        Run the following command to unload the `hfsplus` module:
+        # rmmod hfsplus
+      scored: true
+    - id: 1.1.1.5.b
+      description: "Ensure mounting of hfsplus filesystems is disabled"
+      audit: "lsmod | grep hfsplus"
+      tests:
+        test_items:
+        - flag: ""
+          compare:
+            op: eq
+            value: ""
+          set: true
+      remediation: |
+        Edit or create the file `/etc/modprobe.d/CIS.conf` and add the following line:
+        install hfsplus /bin/true
+        Run the following command to unload the `hfsplus` module:
+        # rmmod hfsplus
+      scored: true
+    - id: 1.1.1.6.a
+      description: "Ensure mounting of squashfs filesystems is disabled"
+      sub_checks:
+      - check:
+          audit: "modprobe -n -v squashfs"
+          constraints:
```

Comment:
Do you know why we have constraints for RHEL7 for some of these tests? I don't see anything in the CIS spec to explain why we have that. 

Same question for 1.1.1.6.b, and 1.1.1.8.a/b

Reply:
@yoavrotems

Reply:
Yes, because it couldn't be remediated in other systems like in Ubuntu you will get next output:
> rmmod: ERROR: Module squashfs is builtin.
But should recheck because I believe that it does apply the rhel6